pkg/chrootarchive/chroot_linux.go

```diff
@@ -30,9 +30,11 @@ func chroot(path string) (err error) {
 	if err := mount.MakeRPrivate("/"); err != nil {
 		return err
 	}
-	// ensure path is a mountpoint
-	if err := mount.MakePrivate(path); err != nil {
```

Comment:
`MakePrivate` does a `bind` only (rather than `rbind`, and we don't need to `MakePrivate` anymore sense `/` is set to `rprivate` above... but we do need to make sure the path is a mounted path otherwise pivot_root will fail.